readme.md

```diff
@@ -84,7 +84,11 @@ But all the idea with `Find!` is to use the following `!keywords`
 
 | Action keyword  | site                                            |
 | ---             | ---                                             |
-| +r4             | radio4000.com add a new track from URL          |
+| +r4 [url]       | radio4000.com add a new track from URL          |
+| +draw [title]   | open a new drawing In Google Drive Draw         |
+| +doc [title]    | open a new Google Docs document                 |
+| +sheet          | open a new Google Spreadsheets document         |
```

Comment:
sheet also supports title no?

Reply:
nope, it is added to the url to prevent that the parameter added by the user ends up concatenated in the url

Reply:
it has to be fixed...